lib/src/main/kotlin/com/lemonappdev/konsist/api/ext/sequence/KoClassDeclarationSequenceExt.kt

```diff
@@ -210,15 +215,35 @@ fun Sequence<KoClassDeclaration>.withSomeParents(name: String, vararg names: Str
 }
 
 /**
- * Sequence containing all classes that don't have parents.
+ * Sequence containing all classes that have no parent.
+ *
+ * @return A sequence containing classes that have no parent.
```

Comment:
..no parent - class does not extend any class and does not implement any interface.